giraffe/README.md

```diff
@@ -374,27 +374,61 @@ When using the comma separated values (CSV) from the Flux query as the `fluxResp
+
+- **staticLegend**: _Object. Optional._ An object with properties that override the **legend** properties and apply only to the static legend that is always visible. Has a few properties that are unique to the static legend. Properties are:
+
+  - **backgroundColor**: _string. Optional. Defaults to #0f0e15 when excluded._ The [_CSS color value_](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) of the background in the static legend.
+
+  - **border**: _string. Optional._ The [_CSS border_](https://developer.mozilla.org/en-US/docs/Web/CSS/border) value for the styling of the border around the static legend.
+
+  - **colorizeRows**: _boolean. Optional. Defaults to true when excluded._ Toggles the use of colors for the rows in the static legend. When _true_ the rows will use colors from the color scheme in the rendered graph. When _false_ the rows will use the **fontBrightColor** and include small dots of color in the color scheme of the rendered graph.
+
+  - **columns**: _array[string, ...]. Optional._ When included, this array will determine which column names that should be included in the static legend. If this option is included as an empty array, the static legend will be empty.
+
+  - **cursor**: _string | Object. Optional._ The [_CSS cursor property_](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) of the mouse when inside the area of the static legend.
+
+  - **disable**: _boolean. Optional. Defaults to false when not included._ _true_ means the static legend will not be rendered. _false_ means the static legend will be rendered.
+
+  - **font**: _string. Optional. Defaults to '10px monospace' when excluded._ The [_CSS font_](https://developer.mozilla.org/en-US/docs/Web/CSS/font) value for the styling of the static legend.
+
+  - **fontBrightColor**: _string. Optional. Defaults to #f6f6f8 when excluded._ The [_CSS color value_](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) of any text that is not a column header or in a row inside the static legend.
+
+  - **fontColor**: _string. Optional. Defaults to #bec2cc when excluded._ The [_CSS color value_](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) of the column headers in the static legend. The rest of the static legend will use the color scheme set by the `LayerConfig`'s `colors` options.
+
+  - **heightRatio**: _number. Optional. Defaults to 0 when not included._ A fraction expressed as a decimal to indicate the height of the static legend which is a retangular box directly beneath the graph and axes. Must be between 0 and 1 inclusive. Values outside of this range are considered 0.
```

Comment:
this should either:
default to .2 ;
or be required.

please add:  must set this to a non-zero number to actually show the static legend.

Reply:
We'll default to 0.2 for now. Agree that 0 doesn't make any sense because we want users to have less configuration to worry about.